visualization/include/pcl/visualization/pcl_visualizer.h

```diff
@@ -1200,7 +1200,19 @@ namespace pcl
         setShapeRenderingProperties (int property, double value,
                                      const std::string &id, int viewport = 0);
 
-        /** \brief Set the rendering properties of a shape (3x values - e.g., RGB)
+        /** \brief Set the rendering properties of a shape (2x values - e.g., LUT minmax values)
+          * \param[in] property the property type
+          * \param[in] val1 the first value to be set
+          * \param[in] val2 the second value to be set
+          * \param[in] id the shape object id
+          * \param[in] viewport the view port where the shape's properties should be modified (default: all)
```

Comment:
Do you mean `viewport`? Yes, it is not used in any of the `setShapeRenderingProperties` or 'setPointCloudRenderingProperties' functions. I am not sure why it is there in the first place. Should we remove it?